src/delivery-options/data/request/fetchFromEndpoint.js

```diff
@@ -53,7 +54,9 @@ const memoizedFetch = memoize(async function fetchFunc(endpoint, options = {}) {
  */
 export const fetchFromEndpoint = async(endpoint, options = {}) => {
   const { response, error } = await memoizedFetch(endpoint, options);
-
+  if (error['errors']['0']['code'] === ERROR_WADDEN_ISLANDS) {
```

Comment:
Dit gaat niet goed, hieronder staat een `if (error)`, dus op dit punt is het mogelijk dat error nog niet bestaat. Dan krijg je een error in de console.